modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/ProfileEpisodeListFragment.kt

```diff
@@ -183,10 +183,12 @@ class ProfileEpisodeListFragment : BaseFragment(), Toolbar.OnMenuItemClickListen
             binding?.toolbar?.isVisible = !isMultiSelecting
             binding?.multiSelectToolbar?.setNavigationIcon(R.drawable.ic_arrow_back)
 
-            if (isMultiSelecting) {
-                trackMultiSelectEntered()
-            } else if (wasMultiSelecting) {
-                trackMultiSelectExited()
+            if ((activity as? FragmentHostListener)?.isUpNextShowing() == false) {
```

Comment:
I added this check to track multi-select entered/ exited events only when up next fragment is not showing. 

The way how singleton `MultiSelectHelper`'s `isMultiSelectingLive` is subscribed and `multiSelectHelper.listener` is set, makes it tricky to avoid these events from getting tracked when the fragment is not visible.